src/studying.md

```diff
@@ -133,6 +133,20 @@ answer. Pressing the <kbd>spacebar</kbd> or <kbd>Enter</kbd> will select **Good*
 See [Deck Options](deck-options.md) and the [FAQ](https://faqs.ankiweb.net/what-spaced-repetition-algorithm.html)
 to learn more about how the algorithm works.
 
+## How to choose the answer button
+
+The sections above explain *what* the answer buttons do, but you may also want to know *how* to choose an answer button intuitively,
+without thinking about the algorithm.
+
+- **Again** should be pressed if your answer is incorrect. If your answer is partially correct, you should be strict with yourself: if it would count as
+a fail outside of Anki, then it counts as a fail in Anki as well.
+
+- **Hard** should be pressed if your answer is correct, but you had doubts about it or if you made a minor mistake. If your answer is 90% correct, press **Hard**.
+
+- **Good** should be pressed if your answer is correct, but it took some mental effort to recall it.
```

Comment:
It doesn't communicate that Good should be used most of the time. I also think we should stick with whatever dae wrote, with little modifications.

Reply:
Dae's recommendations for Hard and Easy aren't great. We shouldn't make the user think about interval lengths or the algorithm.

Reply:
I added a clarification that Good is supposed to be the most commonly used button.